horace_core/sqw/file_io/@dnd_binfile_common/dnd_binfile_common.m

```diff
@@ -455,10 +455,35 @@ function check_obj_initated_properly(obj)
             sqw_obj  = obj.sqw_holder_;
         end
         %
-        function is = is_activated(obj)
-            % Check if the file-accessor is bind with open binary file
+        function is = is_activated(obj, read_or_write)
```

Comment:
should this argument be `mode` and take "single character" `r`, `w`, `rw` alternatives to full words?